opm/input/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquiferCell.cpp

```diff
@@ -51,6 +54,12 @@ namespace Opm {
         } else {
             this->porosity = poro[active_index];
         }
+        constexpr auto MIN_AQ_PORO = 1.0e-6;
```

Comment:
I see the point with numerical comparison. A bit concerned about possible issues if the simulator is instantiated with float, so maybe something like `constexpr auto EPSILON = 2*std::numeric_limits<float>::epsilon(); ` ?

Reply:
Epsilon for float is 1.19209e-07. 

Because `poro` is in double, I think we can use 

```c++
constexpr double EPSILON = std::numeric_limits<float>::epsilon();
```

I do not see the point to use `2*`.

Reply:
Yeah.. you're right - no clear point, but any constant here is a bit arbitrary. I'm starting to think the most reasonable thing here would be to threshold the pore volume, respecting any active MINPV. Can have a go at that..